src/puya/ir/destructure/coalesce_locals.py

```diff
@@ -47,7 +47,7 @@ def coalesce_registers(group_strategy: CoalesceGroupStrategy, sub: models.Subrou
     """
     vla = VariableLifetimeAnalysis.analyze(sub)
 
-    # TODO: this uses a basic definition of interference by looking at live-ranges,
+    # TODO : this uses a basic definition of interference by looking at live-ranges,
```

Comment:
Have a ticket for this, but some of the context might be useful still as a comment for now